metricbeat/docs/modules/sql.asciidoc

```diff
@@ -227,6 +227,51 @@ for each row. For example, this event is created for the first row:
 }
 ----
 
+*Oracle Database Connection Pre-requisites*
+
+To get connected with the Oracle Database ORACLE_SID, ORACLE_BASE, ORACLE_HOME environment variables should be set.
+
+For example: Let’s consider Oracle Database 21c installation using RPM manually by following https://docs.oracle.com/en/database/oracle/oracle-database/21/ladbi/running-rpm-packages-to-install-oracle-database.html[this] link, environment variables should be set as follows:
+    `ORACLE_SID=ORCLCDB`
```

Comment:
What is this `ORACLE_SID`?

Reply:
ORACLE_SID identifies the name of a specific database instance. This information is added to the documentation.

Reply:
Oh ok, is it needed to be configured here? I guess that it can be also configured in the module configuration?

Reply:
Yes, you are right. It is not mandatory and the value set will be over written by the value you provide in the module configuration. I will update that part by removing ORACLE_SID reference